### ISSUE TITLE
Focus management doesn't work properly on the line chart with multiple series 

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#37885]
#### Steps to reproduce:

1. Go to the dashboard insight page
2. Navigate to any insight line chart with more than just 1 line 
3. Try to go through chart points with the tab button 
4. See that you can navigate/access only the first series points. 

#### Expected behavior:
The expected a11y hierarchy is
- chart
   - series
      - points
      
So the focus should also follow that hierarchy

<img width="419" alt="Screenshot 2022-06-28 at 14 55 37" src="https://user-images.githubusercontent.com/18492575/176287880-0f5e5758-5605-4761-a0ff-84635ed79b18.png">

**Comment by sourcegraph-bot-2** [comment]
Heads up @joelkw @felixfbecker @vovakulikov @unclejustin - the "team/code-insights" label was applied to this issue.
